models/demos/t3000/llama2_70b/tt/llama_mlp_optimized.py

```diff
@@ -167,11 +186,11 @@ def decode_forward(self, x: List[tt_lib.tensor.Tensor]) -> List[tt_lib.tensor.Te
             compute_kernel_config=self.model_config["COMPUTE_KERNEL_CONFIG_LOFI"],
         )
 
-        w3_out = ttnn.matmul(
+        w3_out = tt_lib.operations.primary.matmul(
```

Comment:
This should be changed back to `ttnn.matmul`, since this commit comes after the switch to use all ttnn matmul